TranquilWatch Watch App/ContentView.swift

```diff
@@ -0,0 +1,125 @@
+//
+//  ContentView.swift
+//  MyWatchApp Watch App
+//
+//  Created by Victoria Reed on 11/17/22.
+//
+
+import SwiftUI
+//import CoreData
+import HealthKit
+
+struct ContentView: View {
```

Comment:
Can you change the name of this file? Since `ContentView` is already used in the Journal part of the app